### DISCUSSION TITLE
更新改动后注意事项！

### DISCUSSION THREAD

**Discussion by hao123nevergiveup** [share121/inter-knot#762] [其他]
【[具体内容点击此处转bilibili](https://www.bilibili.com/video/BV1GJ411x7h7)】
![niko1](https://github.com/user-attachments/assets/394e763c-16bf-470a-99a4-00fbd3d5d7bf)

**Comment by Senior-Guardian** [comment]
怎么在绳网也会被骗啊（半恼

**Comment by aaa00073** [comment]
确实很有用，也很有意义，希望刚入绳网的萌新能够认真阅读，对维护社区帮助很大
